src/extensions/README.md

```diff
@@ -0,0 +1,23 @@
+# Bramble Extension Loading
+
+Bramble extension loading is done by specifying extensions to be loaded in
+[`src/extensions/bramble-extensions.json`](src/extensions/bramble-extensions.json).
+This is an array of objects with the following form:
+
+```
+{
+        "path": "extensions/default/InlineColorEditor",
+        "copy": [
+            "extensions/default/InlineColorEditor/css/main.css",
+            "extensions/default/InlineColorEditor/img/*.png"
+        ]
+}
+```
+
+Here `path` refers to the path under `src/` where the extension's dir lives.
+The optional `copy` array includes file path globs to be used when copying
+files from `src/` to `dist/` for this extension at build time.  Many extensions
+have no external dependencies, other than the `main.js` file and any modules it
```

Comment:
nit: looking at bramble-extensions.json file only four out fifteen don't have external dependencies, so it might be better to phrase this as the exception rather than the baseline.